configure

```diff
@@ -800,6 +801,7 @@ infodir
 docdir
 oldincludedir
 includedir
+runstatedir
```

Comment:
I wonder where all this 'runstatedir' stuff came from? There are no corresponding changes in `configure.ac` that I can see.

Reply:
I guess I'll try out the vanilla autoconf. We'll see if that helps.

Reply:
Yep, it helped. Thanks again!